src/output.rs

```diff
@@ -274,11 +319,126 @@ mod test {
             ],
             docs: None,
             is_enum: false,
+            supports_derive_default: OnceCell::new(),
         };
         cs.rename();
         assert_eq!(&cs.members[0].name, "jwks_uri");
         assert_eq!(&cs.members[1].name, "jwks_uri_x");
         assert_eq!(&cs.members[2].name, "jwks_uri_x_x");
         assert_eq!(&cs.members[3].name, "jwks_uri_x_x_x");
     }
+
+    #[test]
+    fn can_derive_default() {
+        let containers = vec![
+            Container {
+                name: "Simple".to_string(),
+                level: 1,
+                members: vec![],
+                docs: None,
+                is_enum: false,
+                supports_derive_default: OnceCell::new(),
+            },
```

Comment:
kind of a tiny nit, but similar here to make the tests more readable:

```suggestion
            Container {
                name: "Simple".to_string(),
                level: 1,
                ..Container::default()
            },
```